src/AaveV3ListGhoStewards_20230507/AaveV3ListGhoStewards_20230507_Payload.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+import {AaveV3Ethereum, IACLManager} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {GhoSteward} from 'gho-core/contracts/misc/GhoSteward.sol';
+import {IGhoToken} from 'gho-core/contracts/gho/interfaces/IGhoToken.sol';
+
+library Create2Helpers {
```

Comment:
I do not think the library is needed. My personal preference would be moving these functions to the payload as `internal`.